libs/mrwebrtc/src/interop/device_audio_track_source_interop.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// This is a precompiled header, it must be on its own, followed by a blank
+// line, to prevent clang-format from reordering it with other headers.
+#include "pch.h"
+
+#include "device_audio_track_source_interop.h"
+#include "interop/global_factory.h"
+#include "media/device_audio_track_source.h"
+
+using namespace Microsoft::MixedReality::WebRTC;
+
+mrsResult MRS_CALL mrsDeviceAudioTrackSourceCreate(
+    const mrsLocalAudioDeviceInitConfig* init_config,
+    mrsDeviceAudioTrackSourceHandle* source_handle_out) noexcept {
+  if (!source_handle_out) {
+    RTC_LOG(LS_ERROR) << "Invalid NULL audio track source handle.";
```

Comment:
```suggestion
    RTC_LOG(LS_ERROR) << "Invalid NULL source_handle_out.";
```
"Invalid [...] handle" sounds like _the handle_ is invalid rather than the out pointer to it.